# Complete robot state
Emotion emotion
bool is_speaking
bool is_listening
int32 battery_level  # 0-100
SensorData sensor_data
